src/Data/Integer/Properties.agda

```diff
@@ -175,12 +176,21 @@ drop‿+<+ (+<+ m<n) = m<n
 drop‿-<- : ∀ {m n} → -[1+ m ] < -[1+ n ] → n ℕ.< m
 drop‿-<- (-<- n<m) = n<m
 
++-suc>0 : ∀ {n} → +[1+ n ] > +0
```

Comment:
As this doesn't actually involve the `suc` operator for integers, I think this might be better named `+1+n>0`? or `+1+>0`?